cmd/scheduler/app/options/options.go

```diff
@@ -195,10 +192,10 @@ func (s *ServerOption) ParseCAFiles(decryptFunc DecryptFunc) error {
 	return nil
 }
 
-// Default new and registry a default one
-func Default() *ServerOption {
-	s := NewServerOption()
-	s.AddFlags(pflag.CommandLine)
-	s.RegisterOptions()
-	return s
-}
+//// Default new and registry a default one
+//func Default() *ServerOption {
```

Comment:
Please keep it. It is used in other test packages.

Reply:
> Please keep it. It is used in other test packages.
all right!